yabt/builders/proto.py

```diff
@@ -47,10 +47,12 @@
      ('proto_cmd', PT.list, 'protoc'),
      ('gen_python', PT.bool, True),
      ('gen_cpp', PT.bool, True),
+     ('gen_go', PT.bool, False),
```

Comment:
this will force all the users of ybt to update all images used to run protoc to have the go plugin in them.
Currently we are forcing cpp and cpp but I think both of them comes with the default installation of protobuf lib while go doesn't